pkg/virt-controller/services/multus_annotations.go

```diff
@@ -33,21 +33,37 @@ type multusNetworkAnnotation struct {
 	Namespace     string `json:"namespace"`
 }
 
+type multusNetworkAnnotationPool struct {
+	pool []multusNetworkAnnotation
+}
+
+func (mnap *multusNetworkAnnotationPool) add(multusNetworkAnnotation multusNetworkAnnotation) {
```

Comment:
nit: `mnap` is hard to pronounce (which is funny that I even try when reading). I would just use `p` but this is just an opinionated nit, feel free to ignore.